### PR TITLE
add luap parser

### PR DESCRIPTION
**Pull request by max397574**
This adds the luap parser, a parser for lua patterns by @vhyrro .

### CODE REVIEW

**Comment by max397574** [comment]
I'm not sure about the formatting of the queries.
Tell me if I should change something.

File: lua/nvim-treesitter/parsers.lua
Comment (kyazdani42) [inline lua/nvim-treesitter/parsers.lua:1092]: remove this empty line please

File: queries/lua/injections.scm
Comment (kyazdani42) [inline queries/lua/injections.scm:33]: some spacing would be nice

Comment (max397574) [reply queries/lua/injections.scm]: ok now?

**Review by kyazdani42 — APPROVED** [review]
better thank. Let's wait for another review

**Comment by clason** [comment]
Please look at the test failures; the injections seem to have broken the Lua tests.

**Comment by max397574** [comment]
any idea, why the parser isn't installed?
did I forget to add sth anywhere?

**Comment by clason** [comment]
That's not the error you should look; it fails basic tests ("Tests/Run testes"), so it isn't cached.

**Comment by max397574** [comment]
closing this because I don't have the time and knowledge to fix this

**Pull request closed without merging**
